services/p47-blueapi/README.md

```
@@ -1,32 +1,67 @@
-3. Add the client secret and cookie secret to `templates/secret.yaml`:
+4. If blueapi auth is enabled, add the client secret and cookie secret to `templates/secret.yaml`:
 
   - To obtain the client secret, Create a ticket in [Keycloak Client registration](https://jira.diamond.ac.uk/servicedesk/customer/portal/5/create/176) with  
-    Hostname: `https://p47-blueapi.diamond.ac.uk`
-    Keycloak Environment: Dev/Test and Production
-    ClientId: `{instrument}Blueapi`
-    GPG Public Key: Please follow docs [here](https://dev-guide.diamond.ac.uk/authn/how-tos/request-a-registration-with-keycloak/)
+    - Hostname: `https://p47-blueapi.diamond.ac.uk`
+    - Keycloak Environment: Dev/Test and Production
+    - ClientId: `p47Blueapi`
+    - GPG Public Key: Please follow docs [here](https://dev-guide.diamond.ac.uk/authn/how-tos/request-a-registration-with-keycloak/)
```

markdownlint-cli2 (0.23.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)